index.d.ts

```diff
@@ -81,7 +81,13 @@ export class Registry {
 	 * @param registers The registers you want to merge together
 	 */
 	static merge(registers: Registry[]): Registry;
+	
+	/**
+	 * Global registry
+	 */
+	globalRegistry: Registry;
```

Comment:
The global registry is exported (from the JS and .d.ts files) at the top level as `register`. I'm fine adding this, but i don't know if the original intent was for `Register.globalRegistry` to be private.

If this is added, it should be `static globalRegistry: Registry` though.

Reply:
Applied.

It is a better way to do this https://github.com/jorge07/billing-api/blob/master/tests/ui/http/transaction/get.test.ts#L18?
I basically want to clean the registry before each test so wherever is needed I can probe metrics measure that I need without having a conflict registering a metric that already exists

Reply:
`prom.register.clear();` seems the way to go. Closing as no issue here. Thanks!